app/javascript/controllers/content_list_controller.js

```diff
@@ -30,7 +30,9 @@ export default class extends Controller {
   showPdf(evt) {
     evt.preventDefault()
     const fileUri = evt.currentTarget.dataset.url
-    window.dispatchEvent(new CustomEvent('auth-success', { detail: fileUri }))
+    // window.dispatchEvent(new CustomEvent('auth-success', { detail: fileUri }))
```

Comment:
Do we want to keep this or should be remove it?

Reply:
I will remove.